sdk/keyvault/keyvault-certificates/test/README.md

```diff
@@ -0,0 +1,40 @@
+# Testing
+
+To run our tests, first install the dependencies by running `npm install` (or `rush install`) at the root of the source code of this library (keyvault-certificates).
+With the dependencies installed, you'll be able to run the unit tests and integration tests.
+
+## Unit tests
+
+To run unit tests, go to the root of this library (keyvault-certificates) and run `npm run unit-test`.
+This will run tests in a purely nodejs environment, and in a browser environment using [Karma](https://karma-runner.github.io/latest/index.html).
+These tests aim to reproduce the behavior of our library against remote endpoints through previously recorded HTTP request and responses.
+You won't need to have any environment configuration, nor internet access, for these tests to run.
+
+Keep in mind that if you modify the test files, the API shape of this library, or the underlying network calls, the recordings need to be re-generated.
+You can read more in the section: [Regenerating recordings](#regenerating-recordings).
+
+## Integration tests
+
+Our integration tests will run against the live resources, which are determined by the environment variables you provide.
+To run the integration tests, you will need to execute `npm run integration-test`.
+
+The environment variables needed are:
+
+- `AZURE_CLIENT_ID`: The Client ID of your Azure account.
+- `AZURE_CLIENT_SECRET`: The secret of your Azure account.
+- `AZURE_TENANT_ID`: The Tenant ID of your Azure account.
+- `KEYVAULT_NAME`: The name of the Key Vault you want to run the tests against.
+
+Integration tests won't need updated recordings, and they also won't change previous recordings unless specified (see the section below).
```

Comment:
I prefer links rather than "see the section below".
In the future if we move things around or we add more sections, then links would still apply, but the below/top references might not